cmd/tidb-dashboard/cmd/kvmode_clear.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/pingcap-incubator/tidb-dashboard/pkg/pd"
+	globalUtil "github.com/pingcap-incubator/tidb-dashboard/pkg/utils"
+)
+
+var kvModeAuthClearCmd = &cobra.Command{
+	Use:   "auth-clear",
```

Comment:
how about calling it `revoke`, since kvmode log in is forbidden after doing this operation.